lib/livebook.ex

```diff
@@ -72,6 +72,9 @@ defmodule Livebook do
       ]
   """
 
+  @doc false
+  def app_version, do: Application.spec(:livebook, :vsn) |> List.to_string()
```

Comment:
Let's move this to Livebook.Config or Livebook.Utils perhaps? Since those other modules are private?